overlord/snapstate/snapstate.go

```diff
@@ -1187,6 +1275,7 @@ func doUpdate(ctx context.Context, st *state.State, names []string, updates []*s
 		revnoOpts, flags, snapst := params(update)
 		flags.IsAutoRefresh = globalFlags.IsAutoRefresh
 
+		/*
```

Comment:
we can't drop this, because the flags related settings on a snap might have changed since we checked?